fairseq/models/speech_to_text/berard.py

```diff
@@ -150,7 +150,7 @@ def build_decoder(cls, args, task):
             attention_dim=args.attention_dim,
             output_layer_dim=args.output_layer_dim,
         )
-        if getattr(args, "load_pretrained_decoder_from", None):
+        if getattr(args, "load_pretrained_decoder_from", None) is not None:
```

Comment:
nit: is not None condition check is not needed I guess?
```suggestion
        if getattr(args, "load_pretrained_encoder_from", None):
```